mlir/test/Transforms/lab4/kashirin_alexander/tests.mlir

```diff
@@ -0,0 +1,52 @@
+// RUN: split-file %s %t
+// RUN: mlir-opt -load-pass-plugin=%mlir_lib_dir/KashirinMaxDepthPass%shlibext --pass-pipeline="builtin.module(func.func(KashirinMaxDepthPass))" %t/func1.mlir | FileCheck %t/func1.mlir
+// RUN: mlir-opt -load-pass-plugin=%mlir_lib_dir/KashirinMaxDepthPass%shlibext --pass-pipeline="builtin.module(func.func(KashirinMaxDepthPass))" %t/func2.mlir | FileCheck %t/func2.mlir
+// RUN: mlir-opt -load-pass-plugin=%mlir_lib_dir/KashirinMaxDepthPass%shlibext --pass-pipeline="builtin.module(func.func(KashirinMaxDepthPass))" %t/func3.mlir | FileCheck %t/func3.mlir
+
+//--- func1.mlir
+func.func @func1(%arg0: i32) -> i32 {
```

Comment:
Please add also test case for `llvm.func` type.

Reply:
llvm.func test case added